### PR TITLE
Update "About Us' to include Joel Chan

### LINKED ISSUE #4: Update "About Us" - Joel Chan
Update the about page to include information regarding Joel Chan

### PR DESCRIPTION
**Pull request by JoelChanZhiYang**
Resolves #4

### CODE REVIEW

File: docs/team/joelchan.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Given below are my contributions to the project.
+* **New Feature**: Added the ability for the user to add the status of insurance
+claims to a contact
+* **Project Management**:
+  * Managed issues for milestone v1.1 on GitHub
+* **Documentation**: 
+  * Wrote documentation for claims feature
+* **Community**:
+* **Tools**: 
+    * Added CodeCov integration to the repo
```
Comment (rldefa) [inline docs/team/joelchan.md]: Doesn't matter at all but sliiiightly annoying that this is indented 1 more than the rest.

Comment (rldefa) [inline docs/team/joelchan.md:20]: Does not actually matter but slightly annoying that this is indented one extra time.

**Review by rldefa — APPROVED** [review]
LGTM

File: docs/team/joelchan.md (hunk 1 above)
Comment (JoelChanZhiYang) [reply docs/team/joelchan.md:20]: The indentation is to make a nested list.

Similar to this.
- Outer list
    - Inner list

Comment (rldefa) [reply docs/team/joelchan.md:20]: I am aware, line 20 has 1 more space than line 17 is what I was referring to. Really nitpicking here though feel free to ignore. 👍

**Pull request merged**